### PR TITLE
BoundingBox drawing

### PR DESCRIPTION
**Pull request by ChloeDawn**

### CODE REVIEW

File: mappings/net/minecraft/client/render/Renderer.mapping [hunk 1]
```diff
@@ -93,14 +138,49 @@ CLASS dee net/minecraft/client/render/Renderer
...
+	METHOD b drawShapeOutline (Lcmx;DDDFFFF)V
+		ARG 0 shape
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
+		ARG 7 red
+		ARG 8 green
+		ARG 9 blue
+		ARG 10 alpha
+	METHOD b drawBoxOutline (Lcou;DDDDDDFFFF)V
```
Comment (Gegy) [inline mappings/net/minecraft/client/render/Renderer.mapping]: `buildBoxOutline`

Comment (asiekierka) [reply mappings/net/minecraft/client/render/Renderer.mapping]: It returns void - build?

Comment (Gegy) [reply mappings/net/minecraft/client/render/Renderer.mapping]: It is building into a `BufferBuilder`

Comment (ChloeDawn) [reply mappings/net/minecraft/client/render/Renderer.mapping]: `putBoxOutline` could work too, as it's putting the given box into the given builder.

File: mappings/net/minecraft/client/render/Renderer.mapping [hunk 2]
```diff
@@ -29,12 +40,18 @@ CLASS dee net/minecraft/client/render/Renderer
 	METHOD a onEntityAdded (Lagv;)V
 	METHOD a renderWorldBorder (Lagv;F)V
 		ARG 1 camera
+		ARG 2 partialTicks
```
Comment (Gegy) [inline mappings/net/minecraft/client/render/Renderer.mapping]: I think Fabric is going with `delta`

Comment (ChloeDawn) [reply mappings/net/minecraft/client/render/Renderer.mapping]: Oops, this was one I overlooked, the others I named are `delta`

**Review by Gegy — APPROVED** [review]

**Comment by ChloeDawn** [comment]
Ready for merge 👍

**Review by Prospector — APPROVED** [review]

**Comment by ChloeDawn** [comment]
`drawSegmentedShapeOutline` -> `drawDebugShapeOutline` as per further discussion on Discord.

**Review by SquidDev — APPROVED** [review]

**Pull request merged**